composeApp/build.gradle.kts

```diff
@@ -87,4 +87,9 @@ android {
     dependencies {
         debugImplementation(compose.uiTooling)
     }
-}
\ No newline at end of file
+}
+dependencies {
+    implementation(libs.places)
+    implementation(libs.androidx.core.i18n)
+    implementation(libs.androidx.material)
+}
```

Comment:
You don't add these dependencies here.. Also, why are they added in the first place?